docusaurus.config.js

```diff
@@ -154,7 +161,7 @@ const siteConfig = {
             { label: 'Home', to: '/' },
             { label: 'Link Tag Generator', to: '/link-tag' },
             { label: 'Revshare Generator', to: '/prob-revshare' },
-            { label: 'Exclusive Content Generator', to: '/exclusive-content' },
+            // { label: 'Exclusive Content Generator', to: '/exclusive-content' },
```

Comment:
The logic for this needs more work so we are temporarily disabling this until we can finish up the update because of the deprecation of receipt verification.